crates/diesel_models/src/merchant_account.rs

```diff
@@ -197,6 +197,60 @@ impl MerchantAccount {
     }
 }
 
+// Edits self based on other value
+// Edits other value based on self
+trait Edit<T> {
+    fn edit_self_from_other(&mut self, other: Self);
+    fn edit_value_from_self(self, value:&mut T);
+}
+
+// Option<T> implements Edit for two scenarios.
+// self reference, points to other if other is Some variant
+// value reference, points to internal value of Option if present. 
+impl<T> Edit<T> for Option<T> {
+    fn edit_self_from_other(&mut self, other: Self) {
+        if other.is_some() {
+            *self = other;
+        }
+    }
+    fn edit_value_from_self(self, value:&mut T){
+        if let Some(x)=self{
+            *value=x;
+        }
+    }
+}
+
+impl MerchantAccount {
+    #[cfg(feature = "v1")]
+    pub fn from_update(&mut self, update: MerchantAccountUpdateInternal) {
```

Comment:
One of the primary reasons we use the [struct expression syntax](https://doc.rust-lang.org/reference/expressions/struct-expr.html) in our `apply_changeset` implementations (and explicitly return `MerchantAccount`, `Profile`, etc.) is so that the compiler can throw errors if we forget to include the new fields here. With this implementation, we'd lose out on that benefit.

Reply:
Cool will refactor.

Reply:
Designed with reference to profile.